src/settings-ui/Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -109,6 +109,28 @@
                             <CheckBox x:Uid="PowerLauncher_ClearInputOnLaunch" IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.ClearInputOnLaunch}" Margin="{StaticResource ExpanderSettingMargin}" />
                         </controls:SettingExpander.Content>
                     </controls:SettingExpander>
+                    <controls:SettingExpander IsExpanded="True">
+                        <controls:SettingExpander.Header>
+                            <controls:Setting x:Uid="PowerLauncher_SearchQueryResultsWithDelay" Icon="&#xF182;" Style="{StaticResource ExpanderHeaderSettingStyle}" >
+                                <controls:Setting.ActionContent>
+                                    <ToggleSwitch IsOn="{x:Bind Mode=TwoWay, Path=ViewModel.SearchQueryResultsWithDelay}"/>
+                                </controls:Setting.ActionContent>
+                            </controls:Setting>
+                        </controls:SettingExpander.Header>
+                        <controls:SettingExpander.Content>
+                            <controls:Setting x:Uid="PowerLauncher_SearchInputDelayMs" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.SearchQueryResultsWithDelay}" Style="{StaticResource ExpanderContentSettingStyle}">
+                                <controls:Setting.ActionContent>
+                                    <muxc:NumberBox Minimum="0"
```

Comment:
Should we have a max value? I suggest 300.

Reply:
And we could set a minimum of 25 I think. I can use PT run with a value of 100 as before implementing the delay.

Reply:
Not sure we're here to avoid people shooting themselves on the foot, but some sensible value for Max might make sense. I'm thinking higher, like 1000 (1 second).

Reply:
Then lets use 1000